docs/notebooks/installation.rst

```diff
@@ -1,77 +1,167 @@
+
+- It includes extensions for both standard Jupyter and JupyterLab. 
+- It also installs a CLI for uploading Notebook files.
+- The the neptune-client package is a dependency.
+
+**Cons**
 
-    node -v
+- After the installation, you still have to perform some manual actions in JupyterLab. 
+- This method is not the preferred way of installing extensions in JupyterLab.
 
+**Procedure**
 
-To confirm that you have npm installed you can run this command in your terminal:
+1. In the console, run:
 
 .. code-block:: bash
 
-    npm -v
+    pip install neptune-notebooks
 
-If you do not have them, please refer to: `installation instructions <https://www.npmjs.com/get-npm>`_.
+2. Start JupyterLab by running:
 
-Using command line
-^^^^^^^^^^^^^^^^^^
-Install `neptune-notebooks <https://www.npmjs.com/package/neptune-notebooks>`_ for your JupyterLab. In terminal run:
+.. code-block:: bash
+
+    jupyter lab
+
+3. In the "Build Recommended" notification that is displayed, click **Build**.
+4. In the "Build Complete" notification that is displayed, click **Reload**.
+
+The extension is loaded.
+
+Method 2: Install the JupyterLab extension only through the command line
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This method installs only the extension to JupyterLab.
+
+**Cons**
+
+The neptune-notebooks pip package, neptune-client pip package and extension to the standard Jupyter are not installed.
+
+**Pros**
+
+This method gives you more flexibility. For example, the extension can be running on a 
+different machine than your code that is executed from the Notebook.
+
+**Procedure**
+
+1. In the console, run:
 
 .. code-block:: bash
 
     jupyter labextension install neptune-notebooks
 
-Remember to install Neptune-client, if you did not do so already:
+2. Start JupyterLab by running:
 
 .. code-block:: bash
 
-   pip install neptune-client
+    jupyter lab
 
-Using JupyterLab extension manager (alternative)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+3. In the "Build Recommended" notification that is displayed, click **Build**.
+4. In the "Build Complete" notification that is displayed, click **Reload**.
 
-**Step 1**
 
-Enable extension manager. Select "Commands" (third icon on left panel), then click on "Enable Extension Manager (experimental)"
+Method 3: Install the JupyterLab extension only, using the JupyterLab Extension Manager
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. image:: ../_static/images/notebooks/ext-manager-1.png
-   :target: ../_static/images/notebooks/ext-manager-1.png
-   :alt: enable extension manager
+**Procedure**
 
-**Step 2**
+1. In JupyterLab, enable Extension Manager, by clicking **Settings** > **Enable Extension Manager (experimental)**.
 
-Go to "Extension Manager" (third icon on left panel) and search for "neptune-notebooks". Click "Install", then "Enable".
+.. image:: ../_static/images/notebooks/extension_manager.png
```

Comment:
I think our previous screenshot was better: https://docs.neptune.ai/_static/images/notebooks/ext-manager-1.png
I don't like the error in the background here.

Reply:
I see.
I cropped the picture so that the Error is no longer visible.
Is that OK?